RUFAS/input_manager.py

```diff
@@ -91,3 +91,78 @@ def _load_data(self) -> None:
                                    info_map)
             except Exception as e:
                 raise e
+
+    def _validate_data(self, eager_termination: bool = True) -> bool:
+        """
+        Validates input data.
+
+        Parameters
+        ----------
+        eager_termination : bool, default=True
+            Flag to determine if the process should terminate upon finding invalid data.
+
+        Returns
+        -------
+        bool
+            True if all data is valid; False otherwise.
+        """
+        info_map = {"class": self.__class__.__name__,
+                    "function": self._validate_data.__name__,
+                    }
+        for key, value in self.__pool.items():
+            if not self._validate(key, value):
+                if eager_termination:
+                    om.add_error("Invalid data.", f"Invalid data found: {key} - {value}", info_map)
```

Comment:
1- This message blongs to self._validate.
2- At this stage, it is a warning, it becomes an error when we try to fix it and it fails.
```suggestion
                    om.add_warning("Invalid data", f"Invalid data found: {key=}; {value=}", info_map)
```